include/extractor/profile_properties.hpp

```diff
@@ -12,7 +12,7 @@ struct ProfileProperties
 {
     ProfileProperties()
         : traffic_signal_penalty(0), u_turn_penalty(0), continue_straight_at_waypoint(true),
-          use_turn_restrictions(false), left_hand_driving(false)
+          use_turn_restrictions(false), left_hand_driving(false), max_speed(0)
```

Comment:
didn't we have discussions on this and that a change here would be changing the API? I am thinking of initialiser lists breaking, if we add an additional member.